### PR TITLE
adding opensaml and encorder new orbit poms

### PR DESCRIPTION
**Pull request by hasinthaindrajee**

### CODE REVIEW

File: encoder/1.2.0.wso2v2/pom.xml [hunk 1]
```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (thusithathilina) [inline encoder/1.2.0.wso2v2/pom.xml]: should be 2017

File: encoder/1.2.0.wso2v2/pom.xml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        </dependency>
+    </dependencies>
+
+    <distributionManagement>
+        <repository>
+            <id>wso2.releases</id>
+            <name>WSO2 internal Repository</name>
+            <url>http://maven.wso2.org/nexus/content/repositories/releases/</url>
+        </repository>
+        <snapshotRepository>
```
Comment (thusithathilina) [inline encoder/1.2.0.wso2v2/pom.xml]: Remove snapshot distro

File: opensaml/2.6.4.wso2v4/pom.xml [hunk 3]
```diff
@@ -0,0 +1,362 @@
+<!--
+ ~ Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (thusithathilina) [inline opensaml/2.6.4.wso2v4/pom.xml]: should be 2017

File: opensaml/2.6.4.wso2v4/pom.xml [hunk 4]
```diff
@@ -0,0 +1,362 @@
...
+        <dependency>
+            <groupId>org.opensaml</groupId>
+            <artifactId>opensaml1</artifactId>
+            <version>${opensaml1.version}</version>
+            <optional>true</optional>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.orbit.joda-time</groupId>
+            <artifactId>joda-time</artifactId>
+            <version>${joda-time.orbit.version}</version>
```
Comment (thusithathilina) [inline opensaml/2.6.4.wso2v4/pom.xml]: Shouldn't this also be optional?

File: opensaml/2.6.4.wso2v4/pom.xml [hunk 5]
```diff
@@ -0,0 +1,362 @@
...
+    </dependencies>
+
+    <distributionManagement>
+        <repository>
+            <id>wso2.releases</id>
+            <name>WSO2 internal Repository</name>
+            <url>http://maven.wso2.org/nexus/content/repositories/releases/</url>
+        </repository>
+
+        <snapshotRepository>
```
Comment (thusithathilina) [inline opensaml/2.6.4.wso2v4/pom.xml]: remove snapshot distro

**Comment by hasinthaindrajee** [comment]
Added the requested changes.

**Pull request merged**